### PR TITLE
Add support for TTS

### PR DESCRIPTION
**Pull request by jfalcou**
Add support for the Tiny Test System library from https://jfalcou.github.io/tts/

### CODE REVIEW

File: ports/tts/vcpkg.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "tts",
+  "version-date": "2022-02-21",
```
Comment (Thomas1664) [inline ports/tts/vcpkg.json:3]: Please use `"version": "2.1` instead.

File: ports/tts/portfile.cmake [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    REPO jfalcou/tts
+    REF 2.1
+    SHA512 d774073a91de26f7b0f2ceea25024ec2d8ca2cbbbdc1f82efa44df1ea11b8d9ae53ed4ceab974dcd0f50877a14fdbfb7172054646b24eb94952b4aa70de726f2
+    HEAD_REF main
+)
+
+vcpkg_cmake_configure(SOURCE_PATH "${SOURCE_PATH}")
+
+vcpkg_cmake_install()
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug")
```
Comment (Thomas1664) [inline ports/tts/portfile.cmake:12]: Please add a call to `vcpkg_cmake_config_fixup()` to make sure your config files are installed correctly.

**Review by Thomas1664 — CHANGES_REQUESTED** [review]
Furthermore, your toolchain files inside `/cmake` might be an issue.

**Comment by jfalcou** [comment]
> Furthermore, your toolchain files inside `/cmake` might be an issue.

Oh ? Why ?
SHould I move them elsewhere or ?

File: ports/tts/vcpkg.json (hunk 1 above)
Comment (jfalcou) [reply ports/tts/vcpkg.json]: Done

File: ports/tts/portfile.cmake (hunk 2 above)
Comment (jfalcou) [reply ports/tts/portfile.cmake:13]: done

**Comment by Thomas1664** [comment]
> > Furthermore, your toolchain files inside `/cmake` might be an issue.

Oh ? Why ?
SHould I move them elsewhere or ? 

Because usually it is not required to set `CMAKE_CXX_COMPILER` yourself. Users of your library might want to select different compilers.
You already fixed this by not adding the toolchain directory to `CMAKE_MODULE_PATH` on the master branch. By looking at your latest master branch, I would recomend you use this instead of your latest version tag.

**Comment by jfalcou** [comment]
> > > Furthermore, your toolchain files inside `/cmake` might be an issue.
> 
> Oh ? Why ? SHould I move them elsewhere or ?
> 
> Because usually it is not required to set `CMAKE_CXX_COMPILER` yourself. Users of your library might want to select different compilers. You already fixed this by not adding the toolchain directory to `CMAKE_MODULE_PATH` on the master branch. By looking at your latest master branch, I would recomend you use this instead of your latest version tag.

Well, I don't get it.
Those toolchain files are used for our CI infrastructure. So I don't see how they can interact with w/e an user of the library can do. Especially has it's a header only library.

Now, indeed the version is a bit old. OK, let's close this PR and lemme put up a better version for this integration.
Sorry for th enoise and thanks for the heads up.

**Pull request closed without merging**